trajectory_executor/src/trajectory_executor_node.cpp

```diff
@@ -55,8 +55,8 @@ namespace trajectory_executor
       out[config_.default_control_plugin] = config_.default_control_plugin_topic;
 
       //Hardcoding platoon control plugins
-      std::string control_plugin2 = "platoon_control";
-      std::string control_plugin_topic2 = "/guidance/plugins/platoon_control/plan_trajectory";
+      std::string control_plugin2 = "platooning_control";
+      std::string control_plugin_topic2 = "/guidance/plugins/platooning_control/plan_trajectory";
```

Comment:
From the `CI: Run Tests` CI step, it looks like a unit test failure was introduced here:
```
2024-07-22T20:35:33.9337470Z 1: /opt/carma/src/carma-platform/trajectory_executor/test/test_trajectory_executor.cpp:95: Failure
2024-07-22T20:35:33.9339501Z 1: Expected: (test_suite_node->msg_count) > (0), actual: 0 vs 0
2024-07-22T20:35:33.9340622Z 1: Failed to receive whole trajectory from TrajectoryExecutor node.
2024-07-22T20:35:34.1153518Z 1: [  FAILED  ] TrajectoryExecutorTestSuite.test_emit_traj (3661 ms)
```

I think we will need to update the control topic used for the unit test here: https://github.com/usdot-fhwa-stol/carma-platform/blob/a92c44e692e7f36d0ace654dfc0c793769693ecd/trajectory_executor/test/trajectory_executor_test_suite.cpp#L68